site/en/docs/apps/analytics/index.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Analytics"
-seoTitle: "Google Analytics for Chrome apps"
+seoTitle: "Google Analytics for Chrome apps [Deprecated]"
 date: 2013-08-01
-#updated: TODO
+updated: 2023-06-22
```

Comment:
Not sure the title change is enough to warrant updating `updated`. @jpmedley wdyt?

Reply:
No. Per the handbook, only factual changes to content warrant changing this. I advocate being pretty strict about this because long-term that gives us heuristic for judging whether something is a candidate for updating. (Not an absolute one, I grant you. Just a way to simplify things.)